docs/events.md

```diff
@@ -0,0 +1,77 @@
+## Local Testing of Cloud Events
+
+The setup for cloud functions that accept events is very similar to the instructions in the quickstart, with the following adjustments:
+
+In your `package.json`, add a signature type (in bold) to your start command:
```

Comment:
Nit: remove? (Maybe italicize _signature type_ instead?)

Reply:
Bolded sig type instead as it is important.